cmd/xl-v1-healing_test.go

```diff
@@ -333,8 +333,12 @@ func TestQuickHeal(t *testing.T) {
 		}
 	}
 
+	// figure out read and write quorum
+	readQuorum := len(xl.storageDisks) / 2
+	writeQuorum := len(xl.storageDisks)/2 + 1
+
 	// Heal the missing buckets.
-	if err = quickHeal(xl.storageDisks, xl.writeQuorum, xl.readQuorum); err != nil {
+	if err = quickHeal(*xl, writeQuorum, readQuorum); err != nil {
```

Comment:
You could simply take `xl` directly and change the `quickHeal(xlObj *xlObjects ...`